app/src/main/java/com/vero/tasky/auth/util/PasswordValidator.kt

```diff
@@ -3,28 +3,26 @@ package com.vero.tasky.auth.util
 import com.vero.tasky.R
 import com.vero.tasky.auth.domain.usecase.password.PasswordValidationResult
 
-object PasswordErrorHandler {
+object PasswordValidator {
```

Comment:
Mmmmm this object isnt in charge of validating the password
It just parses the error, and converts it to a string

I liked the ErrorHandler name a bit better
Or maybe
PasswordErrorMapper